__tests__/app.test.js

```diff
@@ -228,7 +228,7 @@ describe("POST /api/articles/:article_id/comments", () => {
 });
 
 describe("PATCH:/api/articles/:article_id", () => {
-  test("202: Responds with the updated article with ammended votes property", () => {
+  test("202: Responds with a status 202 and the updated article with ammended votes property", () => {
```

Comment:
A 202 means the server has accepted the request as valid, but not yet either started or finished processing it yet (like asking an alarm to sound in 20 mins). As our controller and model deals with the request before a response is sent, 202 isn't appropriate here! What could we go for?